app/src/main/java/com/keylesspalace/tusky/util/SpanUtils.kt

```diff
@@ -59,22 +58,39 @@ private fun <T> clearSpans(text: Spannable, spanClass: Class<T>) {
 }
 
 private fun findPattern(string: String, fromIndex: Int): FindCharsResult {
-    val result = FindCharsResult()
+    var foundResult: FindCharsResult? = null
     for (i in fromIndex..string.lastIndex) {
         val c = string[i]
         for (matchType in FoundMatchType.values()) {
```

Comment:
```suggestion
        for ((matchType, finder) in finders) {
```

Reply:
I know it was there before but if you're at it already...

Reply:
Hm, we specifically wanted to iterate in enum value order before, but now it doesn't matter

Reply:
It was so implicit that maybe it was not the most expressive way to write it anyway